app/src/main/java/com/realityexpander/tasky/agenda_feature/data/repositories/eventRepository/remote/eventApi/DTOs/EventDTO.kt

```diff
@@ -67,7 +68,7 @@ abstract class EventDTO {
 
         val host: UserId,
         val isUserEventCreator: Boolean,
-        val isGoing: Boolean? = null,
+        val isGoing: Boolean? = true,
```

Comment:
Is there any case where this would be `null`? If not, you can as well make it non-nullable

Reply:
OK will use non-nullable